src/pages/setting/components/modal/MemberInviteModal.tsx

```diff
@@ -1,18 +1,37 @@
 import IcX from '../../../../assets/icons/x.svg';
 import CopyToClipboard from '../../../../components/Onboarding/CopyToClipboard.tsx';
-import { useRef } from 'react';
+import { useEffect, useRef, useState } from 'react';
 import { createPortal } from 'react-dom';
 import ModalButton from './ModalButton.tsx';
+import { getInviteMembers } from '../../../../apis/setting/useGetInviteMembers.ts';
 
 interface MemberInviteModalProps {
-  memberName: string;
-  url: string;
-  password: string;
   onClick: () => void;
 }
 
-const MemberInviteModal = (props: MemberInviteModalProps) => {
+const MemberInviteModal = ({ onClick }: MemberInviteModalProps) => {
   const inputRef = useRef<HTMLTextAreaElement>(null);
+  const [inviteData, setInviteData] = useState<{
+    name: string;
+    inviteUrl: string;
+    invitePassword: string;
+  } | null>(null);
+
+  useEffect(() => {
+    const fetchInvite = async () => {
+      try {
+        const data = await getInviteMembers();
+        const inviteData = data.result;
+        console.log(inviteData);
+        setInviteData(inviteData ?? null);
+      } catch (err) {
+        console.error(err);
+      }
+    };
+
+    fetchInvite();
+  }, []);
```

Comment:
```suggestion
const { data: inviteMembers } = useGetInviteMembers();
  const [inviteData, setInviteData] = useState<{
    name: string;
    inviteUrl: string;
    invitePassword: string;
  } | null>(null);

  useEffect(() => {
    if (inviteMembers?.result) {
      setInviteData({
        name: inviteMembers.result.name,
        inviteUrl: inviteMembers.result.inviteUrl,
        invitePassword: inviteMembers.result.invitePassword,
      });
    }
  }, [inviteMembers]);
```

P1: useGetInviteMembers를 사용해서 바꿀 수 있어요!!

Reply:
넵 수정하겠습니다!!